src/selectors/getCurrentProjectPreview.js

```diff
@@ -0,0 +1,8 @@
+import makeGetProjectPreview from './makeGetProjectPreview';
+import getCurrentProjectKey from './getCurrentProjectKey';
+
+export default function getCurrentProjectPreview(state) {
+  const getProjectPreview = makeGetProjectPreview();
```

Comment:
I don’t think we want to make a new `getProjectPreview()` function each time `getCurrentProjectPreview()` is called. Instead, I would move this line outside the function so that we are working with the same `getProjectPreview()` function each time.

Since in this case the argument we are passing to `getProjectPreview()` will be the current project key, having the single memoized selector will have the cache behavior we expect/want.